app/com/linkedin/drelephant/analysis/AnalyticJob.java

```diff
@@ -40,8 +40,8 @@
   private static final Logger logger = Logger.getLogger(AnalyticJob.class);
 
   private static final String UNKNOWN_JOB_TYPE = "Unknown";   // The default job type when the data matches nothing.
-  private static final int _RETRY_LIMIT = 3;                  // Number of times a job needs to be tried before going into second retry queue
-  private static final int _SECOND_RETRY_LIMIT = 5;           // Number of times a job needs to be tried before dropping
+  private static final int _RETRY_LIMIT = 1;                  // Number of times a job needs to be tried before going into second retry queue
```

Comment:
why reducing?

Reply:
@aryaKetan Actually, we wanted to make these numbers configurable but since this class implements serializable, its a bit difficult to do that. This is in Todo list for dr-elephant. But for now these are too many retries so reducing it to 2 retries, 1 immediate and another after certain interval.

Reply:
Lets add a metric where-in we capture no of jobs that are dropped due to retry failure.

Reply:
@aryaKetan We already have such metrics in place. Please refer https://github.com/Flipkart/dr-elephant/blob/master/app/com/linkedin/drelephant/analysis/AnalyticJobGeneratorHadoop2.java#L240